Class7 assignment

rawinslow24 opened the pull request:
Created assignment.html to run assignment1.js

Created a function to run for both input year and current year, and nested second function and switch statement inside of first function. Input Year is per assignment, current year to be fancy :P

rawinslow24 committed:
Changes

rawinslow24 committed:
Added assignment1.js

Kilavagora reviewed (changes requested):
Wrap everything in self-executing function in your js file and use const by default. Also does not Atom has auto-formatter? Convention is to have spaces in the array after commas. The rest looks good :)

Also please do no change existing files, like README :) The rest looks great :+1:

rawinslow24 committed:
Changed assignment.js

Wrapped all code in a self-executing function; placed spaces after every comma in array (I think I found an auto-formatter for Atom, but I haven't tested it yet); made duplicate of README.md and removed notes from original one.
